packages/ansible-language-server/src/ansibleLanguageService.ts

```diff
@@ -441,7 +412,9 @@ export class AnsibleLanguageService {
 
         const result: GenerationResponse = await axiosInstance
           .post("/ai/generations/", {
-            content: content,
+            text,
+            create_outline,
```

Comment:
I called the parameter `createOutline` server side to be consistent with the other endpoints where camelCase is used.